### DISCUSSION TITLE
Link Header Pagination

### DISCUSSION THREAD

**Discussion by tylerclair** [Budibase/budibase#8624] [Ideas]
The API that I use puts its pagination data in a Link Header: https://canvas.instructure.com/doc/api/file.pagination.html. It can indicate the next page using a "rel" tag within the header. I have used the npm package parse-link-headers: https://www.npmjs.com/package/parse-link-header and it does exactly as its named. It would be great if that were an option for pagination.

**Comment by melohagan** [comment]
Raised a feature request here! https://github.com/Budibase/budibase/issues/8628
